### ISSUE TITLE
3D View - Panel Right Click menu - Expose the "reset zoom" operator

### ISSUE THREAD

**Issue by Draise14** [Bforartists/Bforartists#4855]
![image](https://github.com/user-attachments/assets/45bd1e38-b48a-4981-8424-f5ed3565f156)

Right now, you can hold down CTRL+MMB to change zoome and also "Reset" the zoom by pressing HOME. But this is 100% undiscoverable and even veteran users may have not known of this feature. 

We should.. try make it atleast kinda discoverable, and we have a right click menu we could use when right clicking on panels in the 3D View. 

# Task
Expose the "reset view" operator to the right click panel to make it discoverable.

Maybe later we can expose the operator to change zoom with a tooltip or expose the zoom panel operators. 

Info from here:
https://blender.stackexchange.com/questions/18615/how-do-i-reset-ui-panel-zoom-level

**Comment by Blenux** [comment]
Like so?

https://github.com/user-attachments/assets/c61eeba1-4607-4ddd-bf23-6737d0272f8c

**Comment by Draise14** [comment]
Now that this is going, should we get the code in? Maybe an icon? I think there is a reset view icon already. Could work.

**Comment by Blenux** [comment]
oh forgot about this, will do later today.

**Comment by Blenux** [comment]
View Reset icon?
![image](https://github.com/user-attachments/assets/ad0e1cbb-3aa0-4c47-9098-e48e48f10e51)

or

File Refresh icon?
![image](https://github.com/user-attachments/assets/58da0b2d-ec9d-48fa-b5ca-e920c3f36199)

I think the file refresh icon works better no?

**Comment by Draise14** [comment]
Yeah, refresh for reset... or reload. Won't work. I guess a new special icon could be done! Reload with a zoom looking glass.

**Comment by Blenux** [comment]
How about something like this, (tried some other ways but were hard to see)

![image](https://github.com/user-attachments/assets/20f0d747-0e00-4c99-b1ec-6a8a8687b360)

![image](https://github.com/user-attachments/assets/7ccb1f67-c04f-4eaa-afa7-0eebc8daff43)
